velero-plugin-for-microsoft-azure/common.go

```diff
@@ -30,21 +30,41 @@ const (
 	cloudNameEnvVar      = "AZURE_CLOUD_NAME"
 
 	resourceGroupConfigKey = "resourceGroup"
+	credentialsFileKey     = "credentialsFile"
 )
 
-func loadEnv() error {
-	envFile := os.Getenv("AZURE_CREDENTIALS_FILE")
-	if envFile == "" {
+func loadEnvFromCredentialsFile(config map[string]string) error {
```

Comment:
I don't remember if we added a general Velero documentation for this new decision making tree (if file, else env var), or if we should be adding this to each plugin doc. Just wanted to ensure we document this, but does not need to be in this PR if it should go in this repo.

Reply:
I was going to add documentation to the main Velero docs, but thinking about it now, the individual plugins will need doc updates to highlight that they are compatible with this new feature. Thanks for pointing that out!